config/wpcalypso.json

```diff
@@ -128,5 +128,6 @@
 	"olark": {
 		"business_account_id": "3146-815-10-9343",
 		"business_group_id": "3033aa025c1a9648c703eb70132ff220"
-	}
+	},
+	"directly_rtm_widget_environment": "sandbox"
```

Comment:
Those envs should use the default `production` build of Directly, so they'll get it from `_shared.json`